external-crates/move/move-vm/runtime/src/interpreter.rs

```diff
@@ -2390,19 +2395,18 @@ impl Frame {
         }
     }
 
-    fn check_depth_of_type(&self, resolver: &Resolver, ty: &Type) -> PartialVMResult<u64> {
+    fn check_depth_of_type(resolver: &Resolver, ty: &Type) -> PartialVMResult<u64> {
         let Some(max_depth) = resolver
             .loader()
             .vm_config()
             .runtime_limits_config
             .max_value_nest_depth else {
             return Ok(1);
         };
-        self.check_depth_of_type_impl(resolver, ty, 0, max_depth)
+        Self::check_depth_of_type_impl(resolver, ty, 0, max_depth)
```

Comment:
@tzakian any reason why this depended on `self` to begin with?

Reply:
Nope! Fine to make this an associated function :)